torch/onnx/utils.py

```diff
@@ -834,9 +834,19 @@ def _trace_and_get_graph_from_model(model, args):
     # before and after running the model.  Fail fast!
     orig_state_dict_keys = torch.jit._unique_state_dict(model).keys()
 
+    # disable the autocast cache as the autocast cache causes type conversions on parameters to
+    # check a cache, which introduces constant tensors to repace wright and bias into the graph
+    # TODO: https://github.com/pytorch/pytorch/issues/84092
+    prev_autocast_cache_enabled = torch.is_autocast_cache_enabled()
+    if torch.is_autocast_enabled() or torch.is_autocast_cpu_enabled():
```

Comment:
nit: curious if this condition is necessary.

Reply:
I suppose disabling autocast when it is already disabled shoudnt do any harm

Reply:
Done